contents/docs/self-host/deploy/snippets/do-ip-address.mdx

```diff
@@ -0,0 +1,5 @@
+```console
```

Comment:
this actually works for anything that uses nginx (potentially other ingress options too), e.g. our AWS deployment guide has the same; just gcp one is different